problems/getCountriesSortedByPopulation.js

```diff
@@ -7,8 +7,13 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
-
+function getCountriesSortedByPopulation(array) {
+        let newArray = [];
+        for (let i = arr.length - 1; i >= 0; i--) {
+          newArray.push(arr[i]);
+        }
+        return newArray;
+      }
```

Comment:
This currently reverses the order of the elements in the array. We want to sort the array. Look into how to sort objects on MDN.